apps/sam-design-system-site/src/app/formly/formly-input/formly-input.component.html

```diff
@@ -0,0 +1,5 @@
+<form [formGroup]="form" (ngSubmit)="submit(model)">
+    <formly-form [form]="form" [fields]="fields" [model]="model"></formly-form>
+    <button type="submit" class="btn btn-primary submit-button" [disabled]="!form.valid">Submit</button>
+</form>
+{{results | json}}

```

Comment:
The app should not have the sample items directory. It should be in a sample library that should be referenced by lazy loading.